e2e/cypress/tests/integration/enterprise/cloud/billing/self_hosted_pricing_modal_spec.js

```diff
@@ -162,7 +162,7 @@ describe('Self hosted Pricing modal', () => {
         // * Check that professional upgrade button is available
         cy.get('#pricingModal').should('be.visible');
         cy.get('#professional').should('be.visible');
-        cy.get('#professional_action').should('not.be.disabled').contains('Upgrade');
```

Comment:
I thought all these needed upgrading at first, but reading more closely, looks like only the first one (#UpgradeButton) needs to be update. The other ones are for a #professional_action, which didn't change.

Reply:
Oh, my bad. I should be paying more attention. I just did a search and replace.